homeassistant/components/recorder/models.py

```diff
@@ -198,6 +210,37 @@ def to_native(self, validate_entity_id=True):
             return None
 
 
+class Statistics(Base):  # type: ignore
+    """Statistics."""
+
+    __table_args__ = {
+        "mysql_default_charset": "utf8mb4",
+        "mysql_collate": "utf8mb4_unicode_ci",
+    }
+    __tablename__ = TABLE_STATISTICS
+    id = Column(Integer, primary_key=True)
+    entity_id = Column(String(255))
+    period = Column(Enum(*STATISTIC_PERIODS))
+    start = Column(DATETIME_TYPE, index=True)
+    end = Column(DATETIME_TYPE, index=True)  # Remove, enough with start time + period?
```

Comment:
As the comment suggests, maybe this can be removed?

Reply:
It's always easier to add it later since SQLite can't drop

Reply:
Fixed in 8b794288a02e7aededbeda47e91624c8180e7995